internal/workflow_testsuite.go

```diff
@@ -170,6 +170,11 @@ func (t *TestActivityEnvironment) SetHeartbeatDetails(details interface{}) {
 	t.impl.setHeartbeatDetails(details)
 }
 
+// SetWorkerStopChannel sets the worker stop channel to be returned from activity.GetWorkerShutdownChannel()
+func (t *TestActivityEnvironment) SetWorkerStopChannel(c chan struct{}) {
```

Comment:
Inconsistent: SetWorkerStopChannel and GetWorkerShutdownChannel. Rename one or another.

Reply:
Make sense. Updated all naming to WokerStopChannel